app/templates/components/forms/orders/order-form.hbs

```diff
@@ -0,0 +1,116 @@
+<form class="ui form" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui yellow inverted segment">
+      <div class="ui inverted small horizontal statistic">
+        <div class="value">
+          8:48
+        </div>
+        <div class="label">
+          {{t 'Please complete registration within 10:00 minutes.'}}
+          <br>
+          {{t 'After 10:00 minutes, the reservation we\'re holding will be released to others.'}}
+        </div>
+      </div>
+    </div>
+    <div class="ui padded segment">
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Buyer'}}
+      </h4>
+      <div class="field">
+        <label class="required" for="firstname">{{t 'First Name'}}</label>
+        {{input type='text' name='first_name' value=buyer.firstName}}
+      </div>
+      <div class="field">
+        <label class="required" for="lastname">{{t 'Last Name'}}</label>
+        {{input type='text' name='last_name' value=buyer.lastName}}
+      </div>
+      <div class="field">
+        <label class="required" for="email">{{t 'Email'}}</label>
+        {{input type='text' name='email' value=buyer.email}}
+      </div>
+      <h4 class="ui horizontal divider header">
+        <i class="ticket icon"></i>
+        {{t 'Ticket Holder\'s Information'}}
+      </h4>
+      {{#each holders as |holder index|}}
+        <div class="inline field">
+          <i class="user icon"></i>
+          <label>{{t 'Ticket Holder '}}{{index}}</label>
+        </div>
+        <div class="field">
+          <label class="required" for="firstname">{{t 'First Name'}}</label>
+          {{input type='text' name=(concat 'first_name_' index) value=holder.firstName}}
+        </div>
+        <div class="field">
+          <label class="required" for="lastname">{{t 'Last Name'}}</label>
+          {{input type='text' name=(concat 'last_name_' index) value=holder.lastName}}
+        </div>
+        <div class="field">
+          <label class="required" for="email">{{t 'Email'}}</label>
+          {{input type='text' name=(concat 'email_' index) value=holder.email}}
+        </div>
+        <div class="field">
+          {{ui-checkbox label=(t 'Same as Ticket Buyer') checked=holder.sameAsBuyer onChange=(action 'fillHolderData' holder)}}
```

Comment:
@geekyd not exactly. This checkbox should be rendered for all by default, but hidden for all once any of it is checked. 

because, multiple ticket types maybe selected. And they can choose to autofill the holder data for the required ticket type.